plugins/woocommerce-blocks/assets/js/blocks/product-filter/types.ts

```diff
@@ -13,6 +13,7 @@ export type FilterType = keyof typeof BLOCK_NAME_MAP;
 export type BlockAttributes = {
 	filterType: FilterType;
 	heading: string;
+	providerNameSlug: string;
```

Comment:
Can you explain why we don't use the existing attribute (`filterType`) and add this new one?

Reply:
I just removed the providerNameSlug, your comment made me realize it is not necessary.